tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
+#          through libyui-rest-client.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use YuiRestClient::Logger;
+
+sub run {
+    # in Disk Activation page press "ZFCP" button
+    my $disk_activation_selector = $testapi::distri->get_disk_activation_selector();
+    $disk_activation_selector->zfcp();
+
+    # in the ZFCP Device overview press the "Add" button
+    my $zfcp_configuration_overview = $testapi::distri->get_zfcp_configuration_overview();
```

Comment:
This two lines should be in another test module, `select_add_new_zfcp_device`. After that you will have another test module `configure_zfcp_device`. You could argue that both cases could be merge in one, but I think we will see it more clear like that IMO.